site/content/installation/ngf-images/jwt-token-docker-secret.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Getting the NGINX Plus Gateway Fabric Image using JWT
+description: "This document describes how to use a JWT token to get an NGINX Plus Gateway Fabric image from the F5 Docker registry."
+weight: 150
+doctypes: ["install"]
+toc: true
+---
+
+## Overview
+
+Follow the steps in this document to pull the NGINX Plus Gateway Fabric image from the F5 Docker registry into your Kubernetes cluster using your JWT token. To list available image tags using the Docker registry API, you will also need to download its certificate and key from [MyF5](https://my.f5.com).
```

Comment:
Anywhere that you have `NGINX Plus Gateway Fabric image`, I would replace with `NGINX Plus image for NGINX Gateway Fabric`. The former makes it sound like that's the product name, which it isn't.